src/ansys/dpf/core/operators/result/magnetic_vector_potential.py

```diff
@@ -126,8 +126,8 @@ def __init__(
         server=None,
     ):
         super().__init__(name="AZ", config=config, server=server)
```

Comment:
Is the `magnetic_vector_potential` operator named `AZ` internally by any chance?
@ricardopeixotocoelho